src/lucky_flow/error_message_when_not_found.cr

```diff
@@ -1,13 +1,20 @@
 class LuckyFlow
   class ErrorMessageWhenNotFound
-    def self.build(selector : String, inner_text : String?, negate : Bool = false)
+    def self.build(selector : String, inner_text : String?, helper : String? =
+                   nil, negate : Bool = false)
```

Comment:
I think this is a bit hard to read. Maybe splitting them onto separate lines?

```
def self.build(
  selector : String,
  inner_text : String?,
  etc.
)
```

And I think you were right. `helper` doesn't need a default